tests/functions/test_visualizations.py

```diff
@@ -163,8 +163,8 @@ def test_npx_clustergrammer_end_to_end(monkeypatch, metadata_df):
 
     # Based on the construction of metadata_df...
     assert col_cats == [
-        ("Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event1"),
-        ("Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event2"),
+        ("CIMAC Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event1"),
+        ("CIMAC Participant ID: CTTTTPP", "Cohort: Arm_A", "Collection Event: Event2"),
```

Comment:
I think shorted col names are better, so I'd suggest even using just "Participant" dropping "cimac" and "id"